### PR TITLE
Give names to our travis jobs so they're easier to identify

### PR DESCRIPTION
**Pull request by ronawho**

### CODE REVIEW

**Comment by ronawho** [comment]
https://travis-ci.org/chapel-lang/chapel/builds/405951985?utm_source=github_status&utm_medium=notification

**Comment by ronawho** [comment]
screen shot 2018-07-19 at 12 15 55 pm

File: .travis.yml [hunk 1]
```diff
@@ -6,12 +6,16 @@ compiler:
 
 matrix:
   include:
-    - env: NIGHTLY_TEST_SETTINGS=true QTHREAD_AFFINITY=no HWLOC_HIDE_ERRORS=1 CHPL_SMOKE_SKIP_DOC=true TEST_COMMAND="./util/buildRelease/smokeTest"
-    - env: NIGHTLY_TEST_SETTINGS=true CHPL_SMOKE_SKIP_MAKE_CHECK=true TEST_COMMAND="./util/buildRelease/smokeTest"
-    - env: TEST_COMMAND="make test-venv && CHPL_HOME=$PWD ./util/run-in-venv.bash ./util/test/check_annotations.py"
+    - name: "make check"
+      env: NIGHTLY_TEST_SETTINGS=true QTHREAD_AFFINITY=no HWLOC_HIDE_ERRORS=1 CHPL_SMOKE_SKIP_DOC=true TEST_COMMAND="./util/buildRelease/smokeTest"
+    - name: "make check-chpldoc"
```
Comment (lydia-duncan) [inline .travis.yml]: I might label this as "includes chpldoc" or the other one as "skips chpldoc".  This label makes it seem like there's a separate script specifically to check that chpldoc builds

File: .travis.yml [hunk 2]
```diff
@@ -6,12 +6,16 @@ compiler:
...
+      env: NIGHTLY_TEST_SETTINGS=true CHPL_SMOKE_SKIP_MAKE_CHECK=true TEST_COMMAND="./util/buildRelease/smokeTest"
+    - name: "check annotations"
+      env: TEST_COMMAND="make test-venv && CHPL_HOME=$PWD ./util/run-in-venv.bash ./util/test/check_annotations.py"
       git:
         depth: 100000
-    - env: TEST_COMMAND="./util/devel/lookForBadRTCalls"
+    - name: "look for bad runtime calls"
```
[the remark this answers is not in the document]
Comment (lydia-duncan) [reply .travis.yml]: I'm ambivalent.  On the one hand, it's nice to know what script might need to be updated right away.  On the other hand, it's visually appealing to have the labels look similar and if there's a failure they're going to check the job output anyways and see what script got run there

File: .travis.yml (hunk 1 above)
Comment (ronawho) [reply .travis.yml]: This one runs the command `make check-chpldoc`, and the other runs the command `make check`

Comment (lydia-duncan) [reply .travis.yml]: If I'm reading smokeTest correctly, the second one runs *both* `make check` and `make check-chpldoc`

Comment (ronawho) [reply .travis.yml]: We set `CHPL_SMOKE_SKIP_MAKE_CHECK` for the second run. It should only be running `make check-chpldoc` (and it looks like `make docs` as well)

Comment (lydia-duncan) [reply .travis.yml]: Oh dur, I was reading the wrong line

Comment (ronawho) [reply .travis.yml]: :). So switch the label to `make check-chpldoc && make docs` and call it good?

Comment (lydia-duncan) [reply .travis.yml]: Works for me!

**Comment by ronawho** [comment]
screen shot 2018-07-19 at 2 56 52 pm

**Pull request merged**